media-libs/libjpeg-turbo/libjpeg_turbo-1.5.3.recipe

```diff
@@ -0,0 +1,149 @@
+early 2010, libjpeg-turbo spun off into an independent project, with the goal \
+of making high-speed JPEG compression/decompression technology available to \
+a broader range of users and developers."
+HOMEPAGE="https://www.libjpeg-turbo.org/"
+COPYRIGHT="1991-2018 Thomas G. Lane, Guido Vollbeding
+	1997-2010 Guido Vollbeding
+	2000-2017 D. R. Commander
+	2009 Bill Allombert
+	1999-2006 MIYASAKA Masaru
+	2011 Siarhei Siamashka
+	2013 Linaro Limited
+	2014 MIPS Technologies, Inc.
+	2015 Google, Inc.
+	2015 Matthieu Darbois"
+LICENSE="BSD (3-clause)
+	IJG
+	Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/libjpeg-turbo/libjpeg-turbo/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="1a17020f859cb12711175a67eab5c71fc1904e04b587046218e36106e07eabde"
+SOURCE_FILENAME="libjpeg-turbo-$portVersion.tar.gz"
+SOURCE_DIR="libjpeg-turbo-$portVersion"
+PATCHES="libjpeg_turbo-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64 arm"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libjpegVersion="62.2.0"
+libjpegVersionCompat="$libjpegVersion compat >= ${libjpegVersion%%.*}"
+libturbojpegVersion="0.1.0"
+libturbojpegVersionCompat="$libturbojpegVersion compat >= ${libturbojpegVersion%%.*}"
+
+PROVIDES="
+	jpeg_turbo$secondaryArchSuffix = $portVersion
+	lib:libjpeg$secondaryArchSuffix = $libjpegVersion
+	lib:libturbojpeg$secondaryArchSuffix = $libturbojpegVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+CONFLICTS="
+	jpeg$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jpeg_turbo${secondaryArchSuffix}_devel = $portVersion
+	devel:libjpeg$secondaryArchSuffix = $libjpegVersion
+	devel:libturbojpeg$secondaryArchSuffix = $libturbojpegVersion
+	"
+REQUIRES_devel="
+	jpeg_turbo$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	jpeg${secondaryArchSuffix}_devel
+	"
+
+PROVIDES_tools="
+	jpeg_turbo${secondaryArchSuffix}_tools = $portVersion
+	cmd:cjpeg$commandSuffix = $portVersion
+	cmd:djpeg$commandSuffix = $portVersion
+	cmd:jpegtran$commandSuffix = $portVersion
+	cmd:rdjpgcom$commandSuffix = $portVersion
+	cmd:tjbench$commandSuffix = $portVersion
+	cmd:wrjpgcom$commandSuffix = $portVersion
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	jpeg_turbo$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_tools="
+	jpeg${secondaryArchSuffix}_tools
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:automake
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:nasm
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage jpeg_turbo$secondaryArchSuffix \
+	$libDir/libjpeg.so.$libjpegVersion \
+	$libDir/libturbojpeg.so.$libturbojpegVersion
+
+BUILD()
+{
+	autoreconf -fi
+	runConfigure --omit-dirs binDir ./configure \
```

Comment:
```diff
-	runConfigure --omit-dirs binDir ./configure \
-		--bindir $commandBinDir \
-		--enable-shared
+	runConfigure ./configure
```
Why? Because `[default=yes]` for `--enable-shared` as we can see with:
```
./configure --help | grep -- --enable-shared
  --enable-shared[=PKGS]  build shared libraries [default=yes]
```